packages/cfd/src/Containers/compare-cfds/compare-cfds.scss

```
@@ -0,0 +1,18 @@
+.compare-cfd-header {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    &-navigation {
+        display: flex;
+        align-items: center;
+        gap: 8px;
+        cursor: pointer;
+    }
+    &-title {
+        flex: 1;
+        &__mobile {
```

You can use mixin here and remove this extra class
see if this works
```suggestion
        @include mobile {
```